CHANGELOG.md

```diff
@@ -5,12 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Legend
+
+[C] Cere Runtime  
+[D] Cere Dev Runtime
+
 ## [vNext]
 
+### Added
+
+- ...
+
 ### Changed
 
 - ...
 
+## [4.8.0]
+
+### Added
```

Comment:
Should we mention that `fast-unstake` is coming to `[C]`?

Reply:
`fast-unstake` is included to update to `polkadot-v0.9.30` It doesn't make sense to duplicate it